### ISSUE TITLE
parsing of multiline properties

### ISSUE THREAD

**Issue by MorphicResonance** [karlicoss/orgparse#52]
Now orgparse unable parse [multiline properties](https://emacs.stackexchange.com/a/38436). Properties where new values/new lines for same items presented as 
```
:PROPERTIES:
:item: value 1
:item+: same item value 2 
:another_item: another value
```
orgparse thinks that they are different items because it works with just single line properties. 
Is it possible to add such thing?

**Comment by karlicoss** [comment]
Yep, can't see why it won't be possible. I don't have a usecase for myself though, and time right now, but will be happy to accept a contribution with tests :)
